conf/zapi/cdot/9.8.0/snapshot.yaml

```diff
@@ -0,0 +1,29 @@
+name:                     Snapshot
+query:                    snapshot-policy-get-iter
+object:                   snapshot
+
+schedule:
+  - instance: 1200s
+  - data: 360s
+
+counters:
+  snapshot-policy-info:
+    - ^^vserver-name     =>  svm
+    - ^^policy           => snapshot_policy
+    - ^comment           => comment
+    - total-schedules    =>  total_schedules
+
+plugins:
+#  SnapmirrorVolume:
+#    refresh_periodicity: 5
+  LabelAgent:
+    replace_regex:
+      - comment comment `^$` `No Comment`
```

Comment:
do we need to change empty comment to `no comment` or does it make more sense to leave empty comments empty? Had a similar question in the plugin code

Reply:
need to check this

Reply:
we don't need to add comment value to 'No Comment' when empty, vserver and policy would serve unique instance key